apps/modernization-ui/src/apps/page-builder/page/management/edit/question/GroupQuestion/GroupQuestion.tsx

```diff
@@ -0,0 +1,150 @@
+import { GroupSubSectionRequest, PagesQuestion, PagesSubSection } from 'apps/page-builder/generated';
+import styles from './group-question.module.scss';
+import { useForm, FormProvider } from 'react-hook-form';
+import { Form, ModalRef } from '@trussworks/react-uswds';
+import { RefObject, useState } from 'react';
+import { ModalToggleButton } from '@trussworks/react-uswds';
+import { RepeatingBlock } from './RepeatingBlock';
+import { SubsectionDetails } from './SubsectionDetails';
+import { SubSectionControllerService } from 'apps/page-builder/generated';
+import { authorization } from 'authorization';
+import { usePageManagement } from '../../../usePageManagement';
+
+type Props = {
+    subsection: PagesSubSection;
+    questions: PagesQuestion[];
+    modalRef: RefObject<ModalRef>;
+};
+
+type Additional = {
+    repeatNumber: number;
+    visibleText: string;
+};
+
+type GroupQuestionFormType = GroupSubSectionRequest & PagesSubSection & Additional;
+
+export const GroupQuestion = ({ subsection, questions, modalRef }: Props) => {
+    const { page, setAlert, refresh } = usePageManagement();
+    const [valid, setValid] = useState(false);
+    const init = {
+        name: subsection.name,
+        batches: questions.map((question) => ({
+            batchTableAppearIndCd: undefined,
+            batchTableColumnWidth: undefined,
+            batchTableHeader: undefined,
+            id: question.id
+        })),
+        blockName: undefined,
+        id: subsection.id,
+        visibleText: subsection.visible ? 'Y' : 'N',
+        repeatNumber: 1
+    };
+    const methods = useForm<GroupQuestionFormType>({
+        mode: 'onChange',
+        defaultValues: { ...init }
+    });
+    const { handleSubmit, reset } = methods;
+
+    const handleGroup = (data: GroupQuestionFormType) => {
+        const request = {
+            batches: data.batches,
+            blockName: data.blockName,
+            id: data.id
+        };
+        try {
+            SubSectionControllerService.groupSubSectionUsingPost({
+                authorization: authorization(),
+                page: page.id,
+                request: request
+            });
+        } catch (error) {
+            if (error instanceof Error) {
+                console.error(error);
+                setAlert({
+                    type: 'error',
+                    expiration: 5000,
+                    message: <p>{error.message}</p>
```

Comment:
Question: are we sure `error.message` is something that will be user friendly?

Reply:
I check to see if it's an intentional error or not, which changes the returned message.
After dealing with the litany of errors, both intentional and otherwise, in create/edit Concept, QA really appreciated that all errors were at least acknowledged with Alert messages. Our designs don't always cover all error-handling so I wanted to push as much information to the user as possible, without reinterpreting the UX myself.